packages/phalanx/src/Phalanx_Field_Def.hpp

```diff
@@ -16,7 +16,12 @@
 #include <vector>
 #include "Teuchos_Assert.hpp"
 #include "Teuchos_TypeNameTraits.hpp"
+#include <Kokkos_Core_fwd.hpp>
+#if !defined(KOKKOS_ENABLE_IMPL_VIEW_LEGACY)
```

Comment:
I'm seeing this macro in a lot of places.  I assume this will go away in some future kokkos release?

Reply:
yes, exactly. Eventually all code protected by the macro (i.e. if the macro is defined) should be deleted.